### PR TITLE
added TransactionRecordMap

### PR DESCRIPTION
**Pull request by fluidvanadium**
Another non-functional change

### CODE REVIEW

File: zingolib/src/wallet/describe.rs [hunk 1]
```diff
@@ -204,7 +205,7 @@ impl LightWallet {
         }
     }
 
-    /// Return a copy of the blocks currently in the wallet, needed to process possible reorgs
+    /// Return a copy of the blocks current.maply in the wallet, needed to process possible reorgs
```
Comment (Oscar-Pepper) [inline zingolib/src/wallet/describe.rs]: classic replace all :)

**Review by Oscar-Pepper — CHANGES_REQUESTED** [review]
The diff wasnt checked, the replace all ran rampant!

Also I have no idea what the purpose of this is?
Current *IS* a map, why does it need a map field?
I assume its to do with seperating a spending from viewing wallet but more context in the description is needed here. It feels like to me there is already a lot of nested unnecessary complexity with Wallet -> Context -> TxMap -> Current where the map of transactions should probably be a direct field of the wallet? This PR adds an extra layer

File: zingolib/src/wallet/transactions/recording.rs [hunk 2]
```diff
@@ -357,7 +367,7 @@ impl TxMapAndMaybeTrees {
         vout: &TxOut,
         output_num: u32,
     ) {
-        // Read or create the current TxId
+        // Read or create the current.map TxId
```
Comment (zancas) [inline zingolib/src/wallet/transactions/recording.rs:370]: Typo?

**Comment by zancas** [comment]
The benefit isn't obvious to me, but haven't tried too hard...

**Review by zancas — COMMENTED** [review]
If this is not necessary to land 317, then let's deprioritize it.

**Review by zancas — CHANGES_REQUESTED** [review]
Deprioritze non-essential work.

**Comment by zancas** [comment]
Oh.. I think I see what's going on here...  I plan to reopen this..

**Comment by zancas** [comment]
Superceded by #911

**Pull request closed without merging**